.ruby-version

```diff
@@ -0,0 +1 @@
+2.1.7
```

Comment:
RVM gives preference to .ruby-version over Gemfile, and so CircleCI does too (if it were not to be the case, it'd fail to load the ruby version, as RVM seds the content of the Gemfile).

On a CI run, then, ruby version comes from .ruby-version. When bundle executes, though, it compares the PATH ruby version (the one defined in .ruby-version) and the one defined in the Gemfile (the dynamic one).

If they mismatch, the run fails.